Core/MemMap.cpp

```diff
@@ -457,4 +477,77 @@ void Memset(const u32 _Address, const u8 _iValue, const u32 _iLength) {
 	CBreakPoints::ExecMemCheck(_Address, true, _iLength, currentMIPS->pc);
 }
 
+bool HandleFault(uintptr_t hostAddress, void *ctx) {
+	SContext *context = (SContext *)ctx;
+	const uint8_t *codePtr = (uint8_t *)(context->CTX_PC);
+
+	// TODO: Check that codePtr is within the current JIT space.
+	bool inJitSpace = MIPSComp::jit->CodeInRange(codePtr);
```

Comment:
Since we install this unconditionally, don't we need to nullcheck `jit`?

-[Unknown]

Reply:
Indeed, will fix that.